Strange test failure in test_biweight

Got the following failure in a [travis run](https://travis-ci.org/astropy/astropy/jobs/386317508) on a PR that has nothing to do with stats. It seems there may be a small rounding error that happened somewhere, but it could also be a real (if small) issue. It is very strange that it wouldn't always happen, given that the random state is initialized the same way...

```
____________________ test_biweight_midcovariance_symmetric _____________________
    def test_biweight_midcovariance_symmetric():
        """
        Regression test to ensure that midcovariance matrix is symmetric
        when ``modify_sample_size=True`` (see #5972).
        """
    
        rng = np.random.RandomState(1)
        d = rng.gamma(2, 2, size=(3, 500))
        cov = biweight_midcovariance(d)
>       assert_equal(cov, cov.T)
astropy/stats/tests/test_biweight.py:215: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
x = array([[ 6.8198942 ,  0.30484383,  0.22600883],
       [ 0.30484383,  6.95537333,  0.58411349],
       [ 0.22600883,  0.58411349,  5.86438159]])
y = array([[ 6.8198942 ,  0.30484383,  0.22600883],
       [ 0.30484383,  6.95537333,  0.58411349],
       [ 0.22600883,  0.58411349,  5.86438159]])
err_msg = '', verbose = True
    def assert_array_equal(x, y, err_msg='', verbose=True):
        """
        Raises an AssertionError if two array_like objects are not equal.
    
        Given two array_like objects, check that the shape is equal and all
        elements of these objects are equal. An exception is raised at
        shape mismatch or conflicting values. In contrast to the standard usage
        in numpy, NaNs are compared like numbers, no assertion is raised if
        both objects have NaNs in the same positions.
    
        The usual caution for verifying equality with floating point numbers is
        advised.
    
        Parameters
        ----------
        x : array_like
            The actual object to check.
        y : array_like
            The desired, expected object.
        err_msg : str, optional
            The error message to be printed in case of failure.
        verbose : bool, optional
            If True, the conflicting values are appended to the error message.
    
        Raises
        ------
        AssertionError
            If actual and desired objects are not equal.
    
        See Also
        --------
        assert_allclose: Compare two array_like objects for equality with desired
                         relative and/or absolute precision.
        assert_array_almost_equal_nulp, assert_array_max_ulp, assert_equal
    
        Examples
        --------
        The first assert does not raise an exception:
    
        >>> np.testing.assert_array_equal([1.0,2.33333,np.nan],
        ...                               [np.exp(0),2.33333, np.nan])
    
        Assert fails with numerical inprecision with floats:
    
        >>> np.testing.assert_array_equal([1.0,np.pi,np.nan],
        ...                               [1, np.sqrt(np.pi)**2, np.nan])
        ...
        <type 'exceptions.ValueError'>:
        AssertionError:
        Arrays are not equal
        <BLANKLINE>
        (mismatch 50.0%)
         x: array([ 1.        ,  3.14159265,         NaN])
         y: array([ 1.        ,  3.14159265,         NaN])
    
        Use `assert_allclose` or one of the nulp (number of floating point values)
        functions for these cases instead:
    
        >>> np.testing.assert_allclose([1.0,np.pi,np.nan],
        ...                            [1, np.sqrt(np.pi)**2, np.nan],
        ...                            rtol=1e-10, atol=0)
    
        """
        assert_array_compare(operator.__eq__, x, y, err_msg=err_msg,
>                            verbose=verbose, header='Arrays are not equal')
E       AssertionError: 
E       Arrays are not equal
E       
E       (mismatch 44.44444444444444%)
E        x: array([[ 6.819894,  0.304844,  0.226009],
E              [ 0.304844,  6.955373,  0.584113],
E              [ 0.226009,  0.584113,  5.864382]])
E        y: array([[ 6.819894,  0.304844,  0.226009],
E              [ 0.304844,  6.955373,  0.584113],
E              [ 0.226009,  0.584113,  5.864382]])
../../../miniconda/envs/test/lib/python3.5/site-packages/numpy/testing/utils.py:787: AssertionError
====== 1 failed, 9975 passed, 808 skipped, 104 xfailed in 229.68 seconds =======
```

--- a/astropy/stats/tests/test_biweight.py
+++ b/astropy/stats/tests/test_biweight.py
@@ -3,7 +3,7 @@
 import pytest
 import numpy as np
 from numpy.random import randn, normal
-from numpy.testing import assert_equal, assert_allclose
+from numpy.testing import assert_allclose, assert_array_almost_equal_nulp
 
 from ..biweight import (biweight_location, biweight_scale,
                         biweight_midvariance, biweight_midcovariance,
@@ -212,10 +212,10 @@ def test_biweight_midcovariance_symmetric():
     rng = np.random.RandomState(1)
     d = rng.gamma(2, 2, size=(3, 500))
     cov = biweight_midcovariance(d)
-    assert_equal(cov, cov.T)
+    assert_array_almost_equal_nulp(cov, cov.T)
 
     cov = biweight_midcovariance(d, modify_sample_size=True)
-    assert_equal(cov, cov.T)
+    assert_array_almost_equal_nulp(cov, cov.T)
 
 
 def test_biweight_midcorrelation():

